DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,14 @@ public function getConfigTreeBuilder()
             ->children()
                 ->scalarNode('db_driver')->isRequired()->end()
                 ->scalarNode('default_context')->isRequired()->end()
+                ->scalarNode('category_manager')
+                    ->info('if sonata-project/classification exists will set "sonata.classification.manager.category" or if you want to define your own category manager you need to implement \Sonata\MediaBundle\Model\CategoryManagerInterface and set the service name here')
```

Comment:
It will be better to shorten the sentence.

This is used for `config:dump-reference` command.